lib/rubocop/cop/minitest/assert_nil.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # This cop checks uses of `assert_equal(nil, ..)`
+    #
+    # @example
+    #   # bad
+    #   assert_equal(nil, actual)
+    #
+    #   # good
+    #   assert_nil(actual)
+    #
+    class AssertNil < Cop
+      MSG = 'Prefer using `assert_nil(..)` over `assert_equal(nil, ..)`.'
```

Comment:
Can you replace the `..` argument with an actual value? The following address is an example.
https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/lint/big_decimal_new.rb#L18-L19